mix.exs

```diff
@@ -63,9 +63,23 @@ defmodule Hexpm.MixProject do
 
   defp aliases() do
     [
+      setup: ["deps.get", "ecto.setup", &setup_yarn/1],
+      "ecto.setup": ["ecto.reset", "run priv/repo/seeds.exs"],
       "ecto.reset": ["ecto.drop", "ecto.create", "ecto.migrate"],
-      "ecto.setup": ["ecto.drop", "ecto.create", "ecto.migrate", "run priv/repo/seeds.exs"],
-      test: ["ecto.migrate", "test"]
+      test: ["ecto.create --quiet", "ecto.migrate", "test"]
     ]
   end
+
+  defp setup_yarn(_) do
+    cmd("yarn", ["install"], cd: "assets")
```

Comment:
@ericmj I just tried hexpm on fresh computer, I was missing `yarn` installed globally (or locally for that matter) and this step crashed; what do you think is the best way to solve it? Maybe we should `npm install` instead (recent npm versions are ok, they have lock etc)

or maybe we should:

```
cmd("npm", ~w(install yarn)", cd: "assets")
cmd("node_modules/.bin/yarn", ~w(install)", cd: "assets")
```

I think I have slight preference for the 2nd idea

Reply:
ugh, it's weird though to end up with `yarn` in `package.json` so maybe it should actually stay as is

Reply:
We can use `npm` instead of `yarn` if it is roughly as fast and use a proper lock, tut either way we should document that one of them is required. We can use `System.find_executable/1` here and print a helping message if we are missing the required tool.